src/manage-authors/edit-coauthor.js

```diff
@@ -0,0 +1,29 @@
+const { gitAuthors } = require('../git-authors');
+
+async function editCoauthor([key, ...args]) {
+  const coauthors = gitAuthors();
+  const authorList = await coauthors.read();
+  if (key in authorList.coauthors) {
+    args
+      .filter(arg => arg.includes('='))
```

Comment:
This code can be simplified by passing in the minimist args object instead of only `args._` because this will contain the name and email keys.  For example you could:

```JavaScript
if(args.name){
  authorList.coauthors[key].name = args.name
}
```